.cicd/test.sh

```diff
@@ -11,9 +11,12 @@ if [[ $(uname) == 'Darwin' ]]; then # macOS
 else # Linux
     COMMANDS="$MOUNTED_DIR/$@"
     . $HELPERS_DIR/file-hash.sh $CICD_DIR/platforms/$PLATFORM_TYPE/$IMAGE_TAG.dockerfile
-    echo "$ docker run --rm --init -v $(pwd):$MOUNTED_DIR $(buildkite-intrinsics) -e JOBS $FULL_TAG bash -c \"$COMMANDS\""
+    mkdir -p $(pwd)/../eosio.contracts/
+    mkdir -p $(pwd)/../eosio.contracts/build/
+    ln -s $(pwd)/build/unittests/contracts/ $(pwd)/../eosio.contracts/build/contracts
+    echo "$ docker run --rm --init -v $(pwd):$MOUNTED_DIR -v $(pwd)/../eosio.contracts/:$MOUNTED_DIR/../eosio.contracts/ $(buildkite-intrinsics) -e JOBS $FULL_TAG bash -c \"$COMMANDS\""
     set +e # defer error handling to end
-    eval docker run --rm --init -v $(pwd):$MOUNTED_DIR $(buildkite-intrinsics) -e JOBS $FULL_TAG bash -c \"$COMMANDS\"
+    eval docker run --rm --init -v $(pwd):$MOUNTED_DIR -v $(pwd)/../eosio.contracts/:$MOUNTED_DIR/../eosio.contracts/ $(buildkite-intrinsics) -e JOBS $FULL_TAG bash -c \"$COMMANDS\"
```

Comment:
I'm not very comfortable with the changes made in this file. Why do the docker commands need to change to support the new tests? Why is this script making new directories and creating symlinks? It doesn't seem like any of this should be necessary to properly run the tests.

Reply:
I would also like to know more about the directories. Regarding the new volume mount argument: `-v $(pwd)/../eosio.contracts:$MOUNTED_DIR/../eosio.contracts`. If this new directory needs to be created, could it be created within the working directory (`$(pwd)/eosio.contracts`)? This would allow for the existing volume mount to capture this change.

Reply:
it was initially required by launcher_service. I've removed this dependency.